pkg/helpers/open/open.go

```diff
@@ -33,7 +33,9 @@ func FindLink(proj *project.Project, linkName string) (url string, err error) {
 		return "", fmt.Errorf("which link should I open?")
 	}
 
-	url = proj.Manifest.Open[linkName]
+	link := project.FindBestLinkMatch(linkName, BuildIndex(proj.Manifest.Open))
+	url = proj.Manifest.Open[link]
```

Comment:
Having a fuzzy search on the custom links **before** the computed links (github, pullrequest) is probably too often disruptive, at least for the shortcut `gh` and `pr`.

It was fine with exact match because it meant that the users would explicitly overload this names.

I think we should evaluate the computed links before.